tap/passive_tapper.go

```diff
@@ -198,8 +201,41 @@ func startMemoryProfiler() {
 	}()
 }
 
+func closeTimedoutTcpStreamChannels() {
+	cpuCount := runtime.NumCPU()
+	// +1 Stats tracker Goroutine
+	// +1 Main packet capture loop
+	baseGoroutineCount := runtime.NumGoroutine() + 2
+	for {
+		runtime.Gosched()
```

Comment:
Why the call to `Gosched`? If this is because this loop doesn't block on anything, then as I understand it this isn't needed in new versions of go (I think 1.16 is included).
Anyway, it is healthier to add a sleep period. This function routine does need to run all the time.

Reply:
@nimrod-up9 yes, since it's like a non-critical housekeeping method basically the loop does not need to run so aggressively. I also thought to add a `time.Sleep` call but couldn't decide the duration. But now I see that `time.Sleep` provides better resource efficiency. So I switched to `time.Sleep` with https://github.com/up9inc/mizu/commit/77eaed598975ef6f0a8bfb550e9b59a61f17a5e8